src/platform/forms-system/src/js/review/widgets.jsx

```diff
@@ -39,6 +39,10 @@ export const yesNo = ({ value, options = {} }) => {
   return <span>{displayValue}</span>;
 };
 
-export const CheckboxWidget = ({ value }) => (
-  <span>{value === true ? 'Selected' : ''}</span>
+export const CheckboxWidget = ({ value, schema = {} }) => (
+  <span>
+    {value === true
+      ? schema.enumNames?.[0] || 'Selected'
+      : schema.enumNames?.[1] || ''}
+  </span>
```

Comment:
I like that `enumNames` is [already part of the system](https://github.com/rjsf-team/react-jsonschema-form/blob/3ec17f1c0ff40401b7a99c5e9891ac2834a1e73f/docs/usage/widgets.md#for-boolean-fields) and we don't have to add another property option to the form config.